### ISSUE TITLE
languages

### ISSUE THREAD

**Issue by Valerian7** [brimdata/brim#1600]
Can languages ​​of other countries be added in the future?

**Comment by philrz** [comment]
@Valerian7: Yes, that's certainly a reasonable request. The app is fairly young and we're still busy adding basic functionality, so it may be a while before the core Dev team is able to make this a priority. But we'll hold the issue open as a reminder and to continue accumulating interest.

**Comment by philrz** [comment]
Another user @cn-fairy asked about this in separate issue https://github.com/brimdata/brim/issues/2371. In their original words:

> Hope to increase the language pack function, for example: to provide a default language pack, the user can facilitate their own needs according to the default language pack language pack file, and the software can load the language pack for changing the language interface. If there is this idea, I can provide help in Simplified Chinese translation.

**Comment by philrz** [comment]
Another community user inquired about this in a [Slack message](https://brimdata.slack.com/archives/C010DR0HHMF/p1678239780862849?thread_ts=1678177258.353039&cid=C010DR0HHMF). In their own words:

> This year there is the idea of internationalization of the UI, before a suggestion of this feature but currently there is no such feature
